src/server/Html.jsx

```diff
@@ -28,7 +28,7 @@ export default class Html extends Component {
       return [
         <script key="vendor" src={assets.javascript.vendor} charSet="UTF-8" />,
         <script key="leaflet" src="//cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-beta.2/leaflet.js" charSet="UTF-8" />,
-	<script key="tangram" src="https://www.nextzen.org/tangram/0.14/tangram.min.js" charSet="UTF-8" />,
+        <script key="tangram" src="https://www.nextzen.org/tangram/0.14/tangram.min.js" charSet="UTF-8" />,
```

Comment:
I just fixed an indentation issue in this file. When trying to santitize calls to dangerouslySetInnerHTML in this file, I got many errors. I am not sure if it's even the best idea to add things here since this is a file that relates to the server.